subprojects/base-services/src/main/java/org/gradle/internal/progress/OperationFinishEvent.java

```diff
@@ -17,7 +17,9 @@
 package org.gradle.internal.progress;
 
 import org.gradle.api.Nullable;
+import org.gradle.internal.scan.UsedByScanPlugin;
 
+@UsedByScanPlugin("Used via InternalTaskExecutionListener")
```

Comment:
@luke @felipefzdz IMO, we should not reference classes that are internal to our build scan plugin in the public source code of Gradle BT.